lib/raop_handlers.h

```diff
@@ -21,7 +21,7 @@
 #include <stdlib.h>
 #include <plist/plist.h>
 
-#define FRAMERATE 15.0
+#define FRAMERATE 15.0 //Framerate
```

Comment:
Why is this a float?

Reply:
Oh
I accidentally changed the repo!
https://github.com/FD-/RPiPlay/pull/190/commits/4a31f9768016c9e5b8642f66fb74d7c776f5314c was the version that I made!
Wait a second reverting

Reply:
np